src/components/common/ListItem.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import styles from './ListItem.module.css'
+import { Button } from './Button'
+
+export const ListItem = ({ id, creditorId, amount, paidAt, users }) => {
+  const [isPaid, setIsPaid] = useState(paidAt)
+
+  const printPrize = (amount) => amount ? amount.toLocaleString('en-US', { minimumFractionDigits: 2, maximumFractionDigits: 2 }).replace('.', ',') : 'N/A'
```

Comment:
You could either move this function outside of your component (or even it's own formatters.js file) or remove the wrapping function as you already have the amount available so there is no need to wrap it again.